tests/behaviour/expectations/behaviour.ts

```diff
@@ -3435,6 +3435,12 @@ export const expectations = flatten(
             new File('derived', 'Derived', [], [Expect.Simple('f', [], ['36', '0', '24', '0'])]),
           ]),
         ]),
+        new Dir('largeSizeArray', [
+          File.Simple('large_size_array', [
+            Expect.Simple('getLength', [], ['100000000', '0']),
+            Expect.Simple('setValue', ['10000000', '0', '456', '0'], ['456', '0']),
```

Comment:
Use utility function `toCairoUint256` to represent the Uint256 inputs. Located in `tests/behaviour/expectations/utils.ts`